src/proxy_app/routing/token_budget.py

```
@@ -0,0 +1,177 @@
+                data = yaml.safe_load(p.read_text()) or {}
+                cdefs = data.get("class_defaults") or {}
+                if isinstance(cdefs, dict):
+                    cfg["class_defaults"].update(cdefs)
+                tm = data.get("tier_multiplier") or {}
+                if isinstance(tm, dict):
+                    cfg["tier_multiplier"].update({str(k): float(v) for k, v in tm.items()})
+                sm = data.get("safety_margin")
+                if isinstance(sm, (int, float)):
+                    cfg["safety_margin"] = int(sm)
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)